pkg/vex/product.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2023 The OpenVEX Authors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vex
+
+// Component abstracts the common construct shared by product and subcomponents
+// allowing OpenVEX statements to point to a piece of software by identifier,
+// hash or identifier.
```

Comment:
This reads a bit off, wondering if we can clarify: `identifier, hash or identifier`

Reply:
> point to a piece of software by referencing it by hash or identifier."